buildpipeline/Core-Setup-Publish.json

```diff
@@ -314,10 +429,23 @@
     },
     "TeamName": {
       "value": "DotNetCore"
+    },
+    "PB_VsoToolsRepo": {
```

Comment:
I think `PB_` prefix is suggested if the variable is consumed in PipeBuild. Same applies to `PB_VsoToolsDir` below.